tools/assets/index.html

```diff
@@ -11,6 +11,7 @@
     <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.7/angular-messages.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.7/angular-aria.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.7/angular-animate.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/d3/4.2.2/d3.min.js"></script>
```

Comment:
Script to load when d3 is excluding from the built app. (during development jspm/systemjs loads the one from "src/jspm_packages".